src/libs/ajax/Apps.js

```diff
@@ -73,5 +73,11 @@ export const Apps = signal => ({
     const res = fetchLeo(`api/apps/v2/${workspaceId}/${appName}`,
       _.mergeAll([authOpts(), jsonBody(body), { signal, method: 'POST' }]))
     return res
+  },
+  deleteAllAppsV2: async workspaceId => {
```

Comment:
cc: IA, these are wiring up the new deleteall endpoints

Reply:
I think it will be necessary to check in with IA about this.

Reply:
I think we'll want to delete the disk by default to avoid leaking them, as well as those of the runtimes, will update.